laptop/setup.sh

```diff
@@ -68,23 +68,38 @@ start_k3s() {
 	mkdir -p ~/.kube/
 
 	cp ./kubeconfig.yaml ~/.kube/config || echo "Failed to get kubeconfig"
+	sed -i 's/127.0.0.1/localhost/g' ~/.kube/config
+	export KUBECONFIG=~/.kube/config
+	cat ~/.kube/config
```

Comment:
Not sure it's "secure" to `cat` this to the terminal 😆

Reply:
you're right removing...

Reply:
removed